buildlib/az-distro-release.yml

```diff
@@ -67,9 +62,13 @@ jobs:
 
       - bash: |
           set -eEx
-          cd pkg-build
-          echo 10 > debian/compat   # https://www.debian.org/doc/manuals/maint-guide/dother.en.html#compat
-          sed -i 's|./contrib|../contrib|' debian/rules  # Hack
+          ./autogen.sh
+          ./contrib/configure-release --with-cuda --with-java=no
```

Comment:
why is there no need for sed hack with contrib path anymore?

Reply:
because the build is no longer out-of-tree on ubuntu